tests/Unit/Models/UserTest.php

```diff
@@ -43,6 +44,14 @@
     expect($user->is_verified)->toBeTrue();
 });
 
+test('prefers questions', function () {
```

Comment:
Doesn't make sense. You're creating a property like true and asserting that true is true. Shouldn't we worry about writing a test validating the update action?

Reply:
fixed